recipes/openal/all/conanfile.py

```diff
@@ -31,12 +31,32 @@ def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
+    @property
+    def _minimum_compiler_supports_cxx14(self):
+        return  {
+            "Visual Studio": "15",
+        }.get(str(self.settings.compiler))
```

Comment:
should this have more compilers?

Reply:
Yes, going to work on it now.
I was on Windows and could not test the Linux compilers.